states/etcd/remove/dirty_importing_segment.go

```diff
@@ -0,0 +1,55 @@
+package remove
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/samber/lo"
+
+	"github.com/milvus-io/birdwatcher/framework"
+	"github.com/milvus-io/birdwatcher/models"
+	"github.com/milvus-io/birdwatcher/states/etcd/common"
+	etcdversion "github.com/milvus-io/birdwatcher/states/etcd/version"
+)
+
+type DirtyImportingSegment struct {
+	framework.ParamBase `use:"remove dirty-importing-segment" desc:"remove dirty importing segments that collection meta already gone"`
+	CollectionID        int64 `name:"collection" default:"0" desc:"collection id to filter with"`
+	Run                 bool  `name:"run" default:"false" desc:"flag to control actually run or dry"`
+}
```

Comment:
Maybe to could add timestamp filter. Only removing segment created before provided timestamp?